### DISCUSSION TITLE
Shortcut control

### DISCUSSION THREAD

**Discussion by insomniachi** [CommunityToolkit/Labs-Windows#456] [Proposals]
bring over shortcut control from power toys
![image](https://github.com/CommunityToolkit/Labs-Windows/assets/6525704/76838fff-e91d-4b74-b21f-d3fd8757ac64)
![image](https://github.com/CommunityToolkit/Labs-Windows/assets/6525704/b7e55d21-1d77-4e56-952c-5163f6dd1bb4)

We should be able to create a KeyboardAccelerator from this control.

**Comment by michael-hawker** [comment]
@insomniachi so to make it as a control that can be used with `SettingsCard` to enable developers to let a user set a shortcut key? And then have something that lets you easily reference/use that set key somewhere else in an app to activate something?

FYI @niels9001 do you know much about this setup in PowerToys or know who does?

**Reply by insomniachi** [reply]
> @insomniachi so to make it as a control that can be used with SettingsCard to enable developers to let a user set a shortcut key? And then have something that lets you easily reference/use that set key somewhere else in an app to activate something?

Yes, exactly.

**Comment by ryanlua** [comment]
Has there been any progress on this proposal? It's a basic accessibility feature that users can remap a key to anything they want whether it be because the keyboard they are using is a different language or the key is unable to be easily pressed.

I want to provide more examples of how others have implemented hotkey/shortcut customization:

###### Windows Terminal

![image](https://github.com/user-attachments/assets/b3eeac3b-a6db-4003-8a79-7cf0eb2f3a09)

##### [Files (third-party)](https://files.community/)

![image](https://github.com/user-attachments/assets/df894e0f-5a08-48bd-8e7c-da73cb4495dd)

**Comment by ReginaldTang** [comment]
Is this going to include a background process to listen for the shortcuts?
